app/lib/ui/pr_card.dart

```diff
@@ -93,23 +93,21 @@ List<Widget> cardContentForSize(BuildContext context, PR pr) {
                 child: Column(
                   crossAxisAlignment: CrossAxisAlignment.start,
                   children: [
-                    Text(
-                      pr.title,
-                      overflow: TextOverflow.ellipsis,
-                      style: const TextStyle(fontWeight: FontWeight.bold),
+                    Row(
+                      children: [
+                        conditionallyColoredDuration(context, pr.created),
+                        Text(
+                          pr.title,
+                          overflow: TextOverflow.ellipsis,
+                          style: const TextStyle(fontWeight: FontWeight.bold),
+                        ),
+                      ]
                     ),
```

Comment:
This breaks the layout on more narrow devices like the Pixel 5.
Mabye instead we could use the `coloredDuration` for the landscape mode and just keep the text (but using timeago description instead) for the portrait layout. Because I don't know what should be wrapping in this current view:

![grafik](https://user-images.githubusercontent.com/39572273/161691791-89cbc62f-1e42-438f-a602-289c7314a656.png)